code/datums/components/remote_materials.dm

```diff
@@ -105,19 +106,31 @@ handles linking back and forth.
 		if (silo == M.buffer)
 			to_chat(user, span_warning("[parent] is already connected to [silo]!"))
 			return COMPONENT_BLOCK_TOOL_ATTACK
+		var/turf/user_loc = get_turf(user)
+		if(GLOB.ore_silo_default == M.buffer && !is_station_level(user_loc.z))
```

Comment:
Since `GLOB.ore_silo_default` only takes into account the roundstart silo, and not replacements, for 3 seconds of work that mining is clearly already willing to put in (and then saddling engineering with the work of actually reconnecting the lathes), it's unlikely this will have the intended effect.

Reply:
Don’t see it as a problem. Ore silo is behind a bolted QM door (and can’t be deconstructed normally) and it is a lot of time/effort  just to be able to bring an ORM down, compared to taking a wrench you spawn next to and taking a machine 2 steps away

Reply:
> Don’t see it as a problem. Ore silo is behind a bolted QM door (and can’t be deconstructed normally) and it is a lot of time/effort just to be able to bring an ORM down, compared to taking a wrench you spawn next to and taking a machine 2 steps away

...The ore silo can be deconstructed just like any other machine? It responds to the standard deconstruction steps and procs like most other machines. And I doubt mining would mind the minor extra work of bothering the AI to open the vault.

Reply:
Sorry it took a while, I wanted to look further into this and just never got around to it, but I have now.

You're right that it can be deconstructed, I thought otherwise from my time playing as QM against swarmers, my memory is probably just terrible.
I didn't see the Shaft Miner issue as much of a problem, because at that point it's sorta an admin-issue if Shaft Miners are breaking into high-risk areas and disconnecting ore silos just because they don't want to spend 30 seconds going to the station to drop mats off, but I changed it anyways because I don't personally think that ghost roles would have machines on separate levels anyways.